types/jquery/index.d.ts

```diff
@@ -146,21 +146,32 @@ interface JQueryStatic {
      * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
      * by passing an HTML string.
      *
-     * @param element_elementArray A DOM element to wrap in a jQuery object.
-     *                             An array containing a set of DOM elements to wrap in a jQuery object.
+     * @param element A DOM element to wrap in a jQuery object.
      * @see \`{@link https://api.jquery.com/jQuery/ }\`
      * @since 1.0
      */
-    <T extends Element>(element_elementArray: T | ArrayLike<T>): JQuery<T>;
+    // tslint:disable-next-line:unified-signatures
+    <T extends Element>(element: T): JQuery<T>;
+    /**
+     * Return a collection of matched elements either found in the DOM based on passed argument(s) or created
+     * by passing an HTML string.
+     *
+     * @param elementArray An array containing a set of DOM elements to wrap in a jQuery object.
+     * @see \`{@link https://api.jquery.com/jQuery/ }\`
+     * @since 1.0
+     */
+    // tslint:disable-next-line:unified-signatures
+    <T extends Element>(elementArray: ArrayLike<T>): JQuery<T>;
```

Comment:
I'm not sure why I made this `ArrayLike` in the first place. The documentation specifies that it accepts an array of elements. Note that changing it to `T[]` seems to fix the issue but triggers a TypeScript 2.4-specific bug that will fail tests.